bugbug/commit_features.py

```diff
@@ -808,34 +809,47 @@ def __call__(self, commit, **kwargs):
 
 
 class Files(object):
+    name = "files"
+
+    def __call__(self, commit, **kwargs):
+        return commit["files"]
+
+
+def _pass_through_tokenizer(doc):
+    return doc
+
+
+class FrequentFilesVectorizer(BaseEstimator, TransformerMixin):
     def __init__(self, min_freq=0.0014):
         self.min_freq = min_freq
+        self.vectorizer = CountVectorizer(
+            tokenizer=_pass_through_tokenizer, lowercase=False
+        )
 
-    def fit(self, commits):
-        self.count = defaultdict(int)
+    def fit(self, X, y=None):
+        min_freq = self.min_freq
```

Comment:
Nit: no need to store it in a temporary variable

Reply:
Removed altogether in b2a4a8e7bc3f5f9141c777edee8a3368a0317ad0.